api/order.go

```diff
@@ -15,6 +16,12 @@ import (
 	"golang.org/x/net/context"
 )
 
+type OrderLineItem struct {
+	SKU      string `json:sku`
+	Path     string `jsonx:"path"`
```

Comment:
Thanks! Amazingly this didn't actually break the functionality since the json decoder just ignored the tag, but still supports lowercase versions of the properties!